src/components/pages/LogIn.tsx

```diff
@@ -403,12 +403,12 @@ class LogInView extends React.Component<LogInProps, LogInState> {
     }
 
     // set the sent time for the resend timer, which occurs when you get to
-    // WAITING_FOR_CONFIRMATION_INPUT state (see the initiatePhoneLogIn action)
+    // WAITING_FOR_CONF_INPUT state (see the initiatePhoneLogIn action)
```

Comment:
I actually prefer more verbose with no disambiguity.. was there a character limit?

Reply:
There is https://firebase.google.com/docs/reference/cpp/group/parameter-names 40 char limit, but I'll go ahead and make ones where we can and undo changes to `PhoneLogInStatus` since we aren't logging that